content/en/docs/contribute/review/for-approvers.md

```diff
@@ -222,4 +222,26 @@ https://github.com/kubernetes/kubernetes/issues.
 If this is a documentation issue, please re-open this issue.
 ```
 
+### Squashing
 
+As an approver, when you review pull requests (PRs), there are various cases where you might do the following:
+- Advise the contributor to squash their commits.
+- Squash the commits for the contributor.
+- Advise the contributor not to squash yet.
+- Prevent squashing.
+
+**Advising contributors to squash**: A new contributor might not know that they should squash commits in their pull requests (PRs). If so, advise them to do so, provide links to useful information, and offer to arrange help if they need it. Some useful links:
```

Comment:
IMO, the previous sentence was a bit confusing.
```suggestion
**Advising contributors to squash**: A new contributor might not know that they should squash commits in their pull requests (PRs). If this is the case, advise them to do so, provide links to useful information, and offer to arrange help if they need it. Some useful links:
```

Reply:
Updated. Thanks, @divya-mohan0209.